contents/docs/deploy.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Deploy PostHog
+sidebar: Docs
+---
+
+PostHog offers a number of different deployment options:
+
+-   [PostHog Cloud](#posthog-cloud) - Create a new PostHog Cloud deployment with just a few clicks.
+    -   Recommended as it's **the fastest way** to try PostHog out.
+    -   Comes with **extra features for reliability and security** and automatic updates.
```

Comment:
> reliability and security

What are these? (maybe a link or remove?)

Reply:
Agreed, slightly tweaked this bullet-point